docs/error-handling.md

```diff
@@ -52,7 +52,7 @@ app.use(async (ctx, next) => {
 
   Error event listeners can be specified with `app.on('error')`.
   If no error listener is specified, a default error listener
-  is used. Error listener receive all errors that make their
+  is used. Error listeners receive all errors that make their
```

Comment:
I this case, it describes the function, which is an error listener "type". I would keep this in singular form IMO.

Reply:
With respect to line 53 I belive plural form makes more sense.